model/plugin_on_install_event.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+// OnInstallEvent is send to the plugin when it gets installed.
+type OnInstallEvent struct {
```

Comment:
do we expect to have more events similar to this in the future - if yes, would it make sense to have an `events` file instead?

Reply:
Yes, I would expect more plugin events to be added. https://github.com/mattermost/mattermost-server/blob/97ccf0bdf6f407d7308d58763f7687d3f079db67/model/plugin_cluster_event.go is the other existing one.

0/5 if it makes sense to consolidate into a single file. They are "stand alone" events and have nothing in common with each other besides that they are events.

Reply:
the idea was to avoid creating one (few lines) file per event if we expect (many) more events in the future.